ui/lib/core/addon/components/replication-dashboard.js

```diff
@@ -27,16 +27,19 @@ export default Component.extend({
     }
     return '';
   }),
-  reindexingProgress: computed('replicationDetails', function() {
-    // TODO: use this value to display a progress bar
-    const { reindex_building_progress, reindex_building_total } = this.replicationDetails;
-    let progress = 0;
+  reindexingProgress: computed(
+    'replicationDetails.{reindex_building_progress,reindex_building_total}',
```

Comment:
i'm not entirely positive this will work if `reindexing_building_progress` or `reindex_building_total` doesn't exist, which is often the case. did you test it out? otherwise i'll work on it in my next pr 🔨

Reply:
I didn't test it so that would be great if you could 🔨 it on your next pr 😁